docs/admin/deployment.md

```diff
@@ -2,6 +2,115 @@
+- **DB_NAME** : Here you must provide the name of the database you setup earlier, which can be found on the “Overview” page of the PostgreSQL add-on. It should be something like “d41vontrqmq2oo”.
+- **AUTH_ROLE** : This is the role of an authenticated user. 
+All authenticated calls to the API will use this user role when attempting to perform actions against the PostgreSQL database. User authentication is done using the procedure described on the Security page in the PostgREST guide, link provided at the bottom. For the purpose of this demo, you can simply input the data in the “User” field of the Overview page of the PostgreSQL add-on. It should be something like “nhvnhzryphzmcp”.
+- **AUTH_PASS** : Simply the password for the authenticated user, also found on the Overview page of the PostgreSQL add-on. It should be something like “5YDLXynT2ZWqUA6c2c_D12TofA”.
+- **ANONYMOUS_ROLE** : This is the role for any unauthenticated call to the API. 
+In a regular PostgreSQL database this role could be for calls that for example don’t alter data, but simply retrieve data. Unfortunately in Heroku Hobby-dev account, it is not possible to add any roles to your PostgreSQL database, which means you will have to use the only role provided when PostgreSQL is setup, the admin role. So input the same data provided in the “AUTH_ROLE” field.
+Please keep in mind that this admin role has full access and can do anything, including deleting from the db.
+- **DB_HOST** : This is the URL to the database, which should be copied directly from the Overview page of the PostgreSQL add-on.
+- **DB_PORT** : The port to used when connecting to the database. Usually not necessary to change unless some specific manual changes have been made to the DB connection.
+- **DB_POOL** : As the description states, “Maximum number of connections in database pool”. This parameter is not necessary to change.
+- **JWT_SECRET** : Information regarding this parameter can be found in the Security link provided. For testing purposes, there is no need to change this parameter, but it is important to read and fully understand how JWT works if and when moving on from testing only.
+- **SCHEMA** : This parameter is very important when it comes to being able to access the information on the PostgreSQL DB via PostgREST, as it indicates what Schema on the DB should be made available through the API calls. Only a single Schema can be selected and the tables and views in that Schema will be accessible through the API, so please make sure that the correct Schema name is input here.
+
+When all the above fields are filled in, the PostgREST configuration should be ready for deployment. Pressing the “Deploy for free” button at the bottom of the screen will initiate the deployment process, which should take no more than a 1-2 minutes to finish.
+![heroku postgrest installing](https://github.com/Mithrandir21/postgrest/blob/master/docs/img/heroku_guide/20_heroku_postgrest_installing.png)<br>
```

Comment:
If you make these images their own paragraph rather than using br are they too far apart?